caikit_nlp/modules/text_generation/peft_tgis_remote.py

```diff
@@ -178,6 +190,7 @@ def save(self, model_path: str):
                     "prompt_cache_id": self._prompt_cache_id,
                     "eos_token": self.eos_token,
                     "verbalizer": self.verbalizer,
+                    # TODO: Add connection_config
```

Comment:
Should the `save()` method not also save the existing connection_config for future use?